app/src/main/java/org/jboss/hal/client/runtime/subsystem/ejb/EjbView.java

```diff
@@ -90,12 +90,14 @@ public EjbView(MetadataRegistry metadataRegistry) {
                 .columns("name","execution-time", "invocations", "wait-time")
                 .build();
 
-        removeMethodsTable = new ModelNodeTable.Builder<>("ejb-bean-remove-methods-table", removeMethodsMetadata)
-                .columns("bean-method", "retain-if-exception")
-                .build();
+        if (removeMethodsMetadata.getDescription().findAttribute(ATTRIBUTES, "bean-method") != null) {
+            removeMethodsTable = new ModelNodeTable.Builder<>("ejb-bean-remove-methods-table", removeMethodsMetadata)
+                    .columns("bean-method", "retain-if-exception")
+                    .build();
+        }
 
         timersTable = new ModelNodeTable.Builder<>("ejb-bean-timers-table", timersMetadata)
-                .columns("info", "time-remaining", "next-timeout", "calendar-timer", "persistent")
```

Comment:
The "info" value-type is not present in the `timers` attribute at all, even in current Wildfly version. I think "info" is only present as an attribute of the "timer-service" child resource, and was added here mistakenly.